WORKSPACE

```diff
@@ -101,26 +101,26 @@ go_repository(
 
 go_repository(
     name = "io_k8s_sigs_kind",
-    commit = "1284e993a84a56c994ea541dbcb97486a7b86b50",
     importpath = "sigs.k8s.io/kind",
+    tag = "0.1.0",
```

Comment:
I'd prefer to track latest SHA (`45ed3d23016727dbdad2f42291ab25e0f09118f1`) on master here. kind is under rapid development and I'd like to advantage of that.
Pinging @BenTheElder to comment.

Reply:
that's a good idea imho, we'll get another tagged release ~next week hopefully, just want to get a few more things in 😅

Reply:
Thanks Ben!

Reply:
Thanks @BenTheElder @justaugustus . Done!